pywbem_mock/_baseprovider.py

```diff
@@ -333,13 +295,114 @@ def remove_namespace(self, namespace):
                 CIM_ERR_NAMESPACE_NOT_EMPTY,
                 _format("Namespace {0!A} contains objects.", namespace))
 
+    @staticmethod
+    def interop_namespace_names():  # pylint: disable=no-self-use
+        """
+        Returns an iterable of the valid interop namespace names where
+        the names are considered case insensitive.
+
+        Returns:
+
+          List of :term:`string` containing the valid names for
```

Comment:
Insert a colon after the type, to make Sphinx recognize the type, like so:
```
List of :term:`string`: List containing ...
```

Reply:
DONE